.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -47,6 +47,7 @@ body:
       description: What version of Kaiden are you running?
       options:
         - "next (development version)"
+      options:
```

YAMLlint (1.38.0)

[error] duplication of key "options" in mapping

(key-duplicates)